src/bucket/BucketManagerImpl.cpp

```diff
@@ -1521,4 +1537,20 @@ BucketManagerImpl::getConfig() const
 {
     return mApp.getConfig();
 }
+
+void
+BucketManagerImpl::reportBucketEntryCountMetrics()
+{
+    if (!mApp.getConfig().isUsingBucketListDB())
+    {
+        return;
+    }
+    auto bucketEntryCounters = mBucketList->sumBucketEntryCounters();
+    for (auto [type, count] : bucketEntryCounters.entryTypeCounts)
+    {
+        mBucketListEntryCountCounters.find(type)->second.set_count(count);
```

Comment:
Can we be more defensive and not potentially deference `end` here if type isn't found? These are just metrics, so we should handle bugs/failures as gracefully as possible, without crashing the node.